### ISSUE TITLE
NVDA does not reliably read content from PDFs in Adobe Acrobat

### ISSUE THREAD

**Issue by w52-ctrl** [nvaccess/nvda#18800]
Problems occur when using NVDA in combination with Adobe Acrobat: the contents of PDF files are not read aloud reliably. We have encountered this problem in several documents, not just in a single case.

Environment:

NVDA versions:

current: 2025.2

Also tested with: 2024.4.2 and 2024.3.1 (same behavior)

Adobe Acrobat version: 2025.001.20630 (latest version)

Operating system: 11 Pro 24H2

Steps to reproduce:

Open PDF file in Adobe Acrobat.

Have NVDA read the text.

Result: Some text is not read aloud or is read incorrectly.

Expected behavior:
NVDA should reliably read aloud the entire content of the PDF document.

Actual behavior:

In several PDF files, text is not read aloud or is only read incompletely.

The problem can be reproduced independently of the specific document.

**Comment by Adriani90** [comment]
This depends on how the pdf document has been created. If the document has the correct accessibility tags, NVDA will read it properly. If not, NVDA will fail.
A pdf document can be created in indesign, in adobe acrobat, or can be exported out of other applications such as MS Word.
Indesign and adobe Acrobat provide the possibility to add the correct accessibility tags to the PDF.
Exporting PDF documents out of Microsoft Office generates automatically accessible PDFs.
Exporting PDF documents from other third party applications is out of control of NVDA, if the third party application does not generate accessible pdf documents.
So please contact the people that created the pdf documents and ask them in which software they were creted. Then, you can decide whether to report it to the software developers or not.
This should be closed as can't fix, since NVDA cannot take care of this work unfortunately. The world is too complex in this regard.

**Comment by SaschaCowley** [comment]
Welcome @w52-ctrl ,

We are unable to process this issue as it stands.

- Please fill out the issue template fully.
- To be able to reproduce this issue, we need a sample document. Provide a minimal PDF document that exhibits this behaviour, preferably with information on how it was authored.
- Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.

Kind Regards,
NV Access Software Developers

**Comment by seanbudd** [comment]
Potential duplicate of #18766

**Comment by seanbudd** [comment]
Closing as we haven't received expected information
